.ci/build

```diff
@@ -37,8 +37,12 @@ source "${SOURCE_PATH}/.ci/common.sh"
 
 ###############################################################################
 
+VERSION="$("${SOURCE_PATH}"/hack/get-version.sh)"
```

Comment:
Maybe you can directly use VERSION file here.

Reply:
To ensure that we capture `*-dev` versions and dirty git trees correctly, it's safer to use `get-version.sh` script

Reply:
Ok, you can resolve this comment.